Resources/Prototypes/Entities/Structures/Decoration/fireplace.yml

```
@@ -46,3 +46,6 @@
           - !type:DoActsBehavior
             acts: [ "Destruction" ]
   - type: AlwaysHot
+  - type: Construction # Frontier
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)